src/index.js

```diff
@@ -26,6 +27,7 @@ class ProcessManager {
   constructor() {
     this.errors = [];
     this.forceShutdown = utils.deferred();
+    this.healthMonitor = new HealthMonitor();
```

Comment:
Should we add a getter for the globalStatus + addCheck to this class?

Reply:
I did it at first, but in a convoluted way that was making use of the `addHook` method, and it became a big mess.

Added it back in a simpler way :smile: